input/pagecontent/smart-web-messaging.md

```diff
@@ -441,19 +446,19 @@ appWindow.postMessage({
 
 | Property              | Optionality | Type   | Description |
 | --------------------- | ----------- | ------ | ----------- |
-| `status`              | REQUIRED    | string | An HTTP response code (i.e. "200 OK" or "404 NOT FOUND"). |
 | `resources`           | OPTIONAL    | object | [FHIR Bundle] containing zero or more resources matching the requested `location`. |
 | `outcome`             | OPTIONAL    | object | [FHIR OperationOutcome] resulting from the message action. |
 {:.grid}
 
-All resources included in the response `resources` Bundle SHALL contain a
-`location` value which matches the value specified in the request `location`
-field; however, if no value was provided in the request, the response Bundle
-SHALL contain the *full* scratchpad contents.
+Because the scratchpad may contain ephemeral objects that are not persisted to
+a FHIR server at all, there are no expectations that a returned bundle entry
+`fullUrl` field references a real resource, or be included in the bundle entry
+at all.
 
-If a `location` was specified in a request but not found in the scratchpad,
-or if there are no entries in the scratchpad, the `resources` property may be
-omitted from the response payload.
+The value of the scratchpad `location` of any Resource accessible through the
+scratchpad can be determined by concatenating the `resourceType` and `id`
+fields, delimiting them with a forward slash.  All resources returned from
```

Comment:
I thought we were adding this "type slash id" explanation to provide a path to getting rid of Bundles -- but the current examples still have Bundles. Is this an intermediate state, or is this as-intended?

Reply:
This is an intermediate state, with the reasoning detailed in [this other comment](https://github.com/HL7/smart-web-messaging/pull/56/files/c3ac78bd5d93a13c93e5dfadb2ffdaa90f08ff48#r686092464).